.eslintrc

```diff
@@ -39,6 +39,7 @@
       2,
       { "args": "after-used", "argsIgnorePattern": "^_", "vars": "local" }
     ],
+    "no-restricted-imports": ["error", "raven"],
```

Comment:
How come eslint changes were necessary?

Reply:
The Raven module for browsers is called `raven-js`, but the Node one is `raven`. I don't want someone to accidentally import the wrong one.

Reply:
Ah good thinking!